components/tidb_query_expr/src/impl_time.rs

```diff
@@ -186,9 +186,9 @@ pub fn sub_datetime_and_duration(
 
 #[rpn_fn(nullable, capture = [ctx])]
 #[inline]
-pub fn from_days(ctx: &mut EvalContext, arg: Option<&Int>) -> Result<Option<Time>> {
+pub fn from_days(ctx: &mut EvalContext, arg: Option<&Int>) -> Result<Option<DateTime>> {
```

Comment:
Ops, `Time` is imported in previous PR, maybe these changes should be split into another PR.